docs/api/features/get.asciidoc

```diff
@@ -12,11 +12,11 @@ Note: You cannot access this endpoint via the Console in {kib}.
 ===== Request
 
 To retrieve all features, issue a GET request to the
-/api/features/v1 endpoint.
+/features/api endpoint.
```

Comment:
__note:__ it feels like not a big deal to change URL in a minor for this particular API.

Reply:
>  Is the /${pluginId}/api format a new standard?

It's not clear yet, that's why I'd like to discuss it here and in https://github.com/elastic/kibana/issues/44620. I need to introduce NP routes in a couple PRs right now and was thinking whether we should separate API and view routes somehow, that's how this `/api/` part  appeared.

> I'd expect this to retrieve a feature with id api, instead of retrieving all registered features.

Heh, that's a good point!

> Maybe /features/api/feature?

Yeah, or something verbose like `/features/api/all-features`, `/features/api/feature/{id}`, still thinking, don't have good enough ideas yet....

I was also thinking about `features/_api/`, `_features/api/`, `features.api/` to kind of stress on the difference between these two semantically different parts of the URL. We can also say that `features/*` are always API calls and `features/{views|app|pages}/*` are for `{views|app|pages}`.

@restrry it's probably a good time to pick some API naming style that we can follow :) Did you guys had a chance to discuss that already?

Reply:
Once https://github.com/elastic/kibana/pull/44855 is merged I'll change URL from `/api/features/v1` to `/api/features`.

Reply:
Changed to `/api/features`